aws/resource_aws_codebuild_project_test.go

```diff
@@ -2896,6 +2928,30 @@ resource "aws_codebuild_project" "test" {
 `, rName, testAccAWSCodeBuildGitHubSourceLocationFromEnv())
 }
 
+func testAccAWSCodeBuildProjectConfig_ARMContainer(rName string) string {
+	return testAccAWSCodeBuildProjectConfig_Base_ServiceRole(rName) + fmt.Sprintf(`
+resource "aws_codebuild_project" "test" {
+  name         = "%s"
+  service_role = "${aws_iam_role.test.arn}"
+
+  artifacts {
+    type = "NO_ARTIFACTS"
+  }
+
+  environment {
+    compute_type = "BUILD_GENERAL1_MEDIUM"
```

Comment:
This test configuration is currently failing:

```
--- FAIL: TestAccAWSCodeBuildProject_ARMContainer (11.15s)
    testing.go:635: Step 0 error: errors during apply:

        Error: Error creating CodeBuild project: InvalidInputException: Compute type BUILD_GENERAL1_MEDIUM is not supported for ARM_CONTAINER projects
```

The [CodeBuild User Guide](https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-compute-types.html) notes that `ARM_CONTAINER` is supported by `BUILD_GENERAL1_LARGE`

```suggestion
    compute_type = "BUILD_GENERAL1_LARGE"
```

Please see the Contributing Guide section on running acceptance testing if you need assistance with that: https://github.com/terraform-providers/terraform-provider-aws/blob/master/.github/CONTRIBUTING.md#running-an-acceptance-test